src/main/java/com/it4045/pokemonviewer/enterprise/service/PokemonServiceStub.java

```diff
@@ -10,7 +11,7 @@
 public class PokemonServiceStub implements IPokemonService{
 
     private final Pokemon pokemon = new Pokemon();
-    private final List<Pokemon> pokemonList = new ArrayList<>();
+    private final Pokedex pokedex = new Pokedex();
```

Comment:
Third Technical Suggestion: Added Pokedex pokedex to replace pokemonList since Pokedex is considered a list for pokemon anyway. Line 31 I added pokedex.getAllPokemon(); to return the list of pokemon.

Reply:
Good observation.  Reduce redundant data!